packages/core/src/fhir-deduplication/shared.ts

```diff
@@ -506,6 +545,27 @@ export function assignMostDescriptiveStatus<T extends Resource & { status?: stri
   target.status = status;
 }
 
+export function assignMostSevereClass(existing: Encounter, target: Encounter): void {
```

Comment:
when possible, let's rename this to have 'dangerously" and all other new mutating functions we're introducing, please